BOT_database_updater.py

```diff
@@ -35,9 +36,12 @@ def process_update_csv(self):
                 self.update_herbarium_code(barcode=row['barcode'], herb_code=row['Modifier'])
 
             # checking lat/long values for update
```

Comment:
for compatibilty between Specify 6 & 7 , now updates require lat/long text to be filled and lat long unit.